docs/json.md

```diff
@@ -154,6 +154,9 @@ If a field is to be declared as "required" in the JSON Schema, the annotation
 ```
 is used.
 
+Please note, we do not recommend using the `$required` annotation for boolean fields, as not setting it is equivalent to `false`.
```

Comment:
I struggled with this formulation. For me it means: We do recommend using the annotation because not setting the annotation is equivalent to `false`. 
Maybe we can have something like 
```suggestion
Please note, we do not recommend using the `$required` annotation for boolean fields, as not setting such a field is equivalent to `false`.
```
to make clear that it is about setting fields and not annotations. Feel free to reformulate my suggestion in better english 😉

Reply:
Do you prefer following suggestion?

```suggestion
"Please note, we do not recommend using the $required annotation for boolean fields, as omitting such a field in the JSON file is equivalent to setting its value to false."
```

Reply:
Yes, I like it

Reply:
Yes, me too. Changed it.